common.gradle

```diff
@@ -34,8 +36,21 @@ android {
         manifestPlaceholders = [
                 appName : "Tachiyomi: $extName",
                 extClass: extClass,
-                nsfw: project.ext.find("isNsfw") ? 1 : 0,
+                nsfw    : project.ext.find("isNsfw") ? 1 : 0,
         ]
+        String baseUrl = project.ext.find("baseUrl") ?: ""
+        if (theme != null && !baseUrl.isEmpty()) {
+            if (baseUrl.endsWith("/")) {
+                println("::warning ::baseUrl[$baseUrl] has a trailing slash")
+            }
+            def split = baseUrl.split("://")
+            assert split.length == 2
```

Comment:
I think build should fail if `baseUrl` contains anything after the domain. so `http://example.com/something` or ``http://example.com/` all fail.

Reply:
What if the site hosts its manga reader in a subdirectory? `https://example.com/something` should be valid IMO.

Reply:
If so, this field should be renamed to `schemeWithDomain` or something. Some sources do require `baseUrl` having more path segments or trailing slash. My intention was to keep this in sync with the one inside the source, and warn if someone updated both to have a trailing slash. This could be moved to some auditing script, etc. I'm open to suggestions.